app/scripts/modules/core/src/application/search/Applications.tsx

```diff
@@ -143,7 +143,7 @@ export class Applications extends React.Component<{}, IApplicationsState> {
     return (
       <div className="infrastructure">
         <div className="infrastructure-section search-header">
-          <div className="container">
+          <div className="container" style={{ overflowY: 'auto' }}>
```

Comment:
do we have a util class that does just this? seems like maybe something worth having.

Reply:
i suppose probably in the style guide is the best place thinking about it, perhaps we should do a pass of more granular utilities for this kind of thing sometime(TM)